app/views/provider_interface/sessions/sign_in_by_email.html.erb

```diff
@@ -0,0 +1 @@
+<%= render 'candidate_interface/shared/check_your_email' %>
```

Comment:
This contains the `try again` link which links to the candidate sign in, right? Do we need to update that?

Reply:
Fixed by copying over the content and changing the link.